src/bench/coin_selection.cpp

```diff
@@ -53,7 +48,13 @@ static void CoinSelection(benchmark::State& state)
         assert(success);
         assert(nValueRet == 1003 * COIN);
         assert(setCoinsRet.size() == 2);
-    }
+
+        // Empty wallet.
+        for (COutput& output : vCoins) {
+            delete output.tx;
+        }
+        vCoins.clear();
```

Comment:
Bit, `std::vector<COutput> vCoins` could be declared inside `do while` to avoid explicit clear.